app/src/main/java/com/epfl/beatlink/ui/library/EditPlaylist.kt

```diff
@@ -193,6 +222,9 @@ fun EditPlaylistScreen(
                           nbTracks = selectedPlaylistState.nbTracks)
                   playlistViewModel.updatePlaylist(updatedPlaylist)
                   playlistViewModel.selectPlaylist(updatedPlaylist)
+                  if (imageUri != Uri.EMPTY) {
+                    playlistViewModel.uploadPlaylistCover(imageUri, context, updatedPlaylist)
+                  }
```

Comment:
Here if we modify the cover image and put a new image, is the new one uploaded ?

Reply:
The new image is uploaded inside of the `permissionLauncher` on line 91